lib/azure/armrest/model/storage_account.rb

```diff
@@ -169,6 +185,20 @@ def copy_blob(src_container, src_blob, dst_container, dst_blob = nil, key = nil)
         Blob.new(response.headers)
       end
 
+      # Delete the given +blob+ found in +container+.
+      #
+      def delete_blob(container, blob, key = nil, options = {})
+        key ||= properties.key1
+
+        url = File.join(properties.primary_endpoints.blob, container, blob)
+        url += "?snapshot=" + options[:date] if options[:date]
+
+        headers = build_headers(url, key, :verb => 'DELETE')
+        response = RestClient.delete(url, headers)
```

Comment:
Feel strange that we still need to explicitly set `:verb => 'DELETE'` since we are calling `RestClient.delete`